test/graphql/groups_query_ransack_test.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class GroupsQueryTest < ActiveSupport::TestCase
+  GROUPS_RANSACK_QUERY = <<~GRAPHQL
+    query($filter: GroupFilter, $orderBy: GroupOrder) {
+      groups(filter: $filter, orderBy: $orderBy) {
+        nodes {
+          name
+          id
+          puid
+          updatedAt
+          createdAt
+        }
+        totalCount
+      }
+    }
+  GRAPHQL
+
+  def setup
+    @user = users(:john_doe)
+  end
+
+  test 'ransack groups query should work' do
+    group = groups(:group_one)
+    original_date = Time.zone.today
+    Timecop.travel(5.days.from_now) do
+      group.created_at = Time.zone.now
+      group.save!
+
+      result = IridaSchema.execute(GROUPS_RANSACK_QUERY,
+                                   context: { current_user: @user },
+                                   variables: { filter: { created_at_gt: (original_date + 1.day).to_s } })
+
+      assert_nil result['errors'], 'should work and have no errors.'
+
+      data = result['data']['groups']['nodes']
+
+      assert_equal 1, data.count
+      assert_equal group.puid, data[0]['puid']
+    end
+  end
+
+  test 'ransack groups query should work with order by' do
```

Comment:
**nitpick (blocking):** Can  you test in here that it is actually returning the groups in the expected order. Maybe the first 2 and the last 2. You could always add a `limit: 3` and just check that the 3 returned results are in the expected order.

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/749/commits/8ae3cfc0e923ffd7c8defb529798b4391a4b8835